tidb-cloud/serverless-export.md

```diff
@@ -0,0 +1,106 @@
+---
+title: TiDB Serverless Export (Beta)
+summary: Learn how to export from TiDB Serverless.
+---
+
+# TiDB Serverless Export (Beta)
+
+TiDB Serverless Export (Beta) is a service that allows you to export data from a TiDB Serverless cluster to local or an external storage service. You can use the exported data for backup, migration, data analysis, or other purposes.
+
+You can also export data with `mysqldump`, `TiDB Dumpling`, or other tools. However, TiDB Serverless Export provides a more convenient and efficient way to export data from a TiDB Serverless cluster. It brings the following benefits:
+
+- Convenience: Export service provides a simple and easy-to-use way to export data from a TiDB Serverless cluster. You don't need additional tools or resources.
+- Isolation: Export service uses separate computing resources, which is isolated from the computing resources used by your online services
+- Consistency: Export service ensures the consistency of the exported data without locking, which will not affect your online services.
+
+## Export to local
+
+When you export to local, there are some limitations:
+
+1. You can only export one table at a time.
+2. The exported data will be expired after two days, please download the data in time.
+3. The exported data will be saved in the stashing area, which offers 250 GB storage space for each organization per region. If the storage space is full, you will not be able to export data to local.
+
+> **Note:**
+>
+> We recommend you export to S3 or other supported storage services if you want to export a large amount of data.
```

Comment:
"other supported storage services" looks unclear. Currently,  besides S3, what storage services are supported for this feature?

Reply:
only s3 is supported now, we will support GCS (Google cloud storage) and Azure Blob Storage next Q